### ISSUE TITLE
TypeError: realpath is not a function

### ISSUE THREAD

**Issue by AlCalzone** [fimbullinter/wotan#604]
I'm not sure if this is a wotan or typescript issue, but it happened during wotan execution on Travis CI:

```
TypeError: realpath is not a function
    at visitDirectory (/home/travis/build/ioBroker/testing/node_modules/typescript/lib/typescript.js:15892:45)
    at Object.matchFiles (/home/travis/build/ioBroker/testing/node_modules/typescript/lib/typescript.js:15888:13)
    at ProjectHost.readDirectory (/home/travis/build/ioBroker/testing/node_modules/@fimbul/wotan/src/project-host.js:32:19)
    at Object.readDirectory (/home/travis/build/ioBroker/testing/node_modules/@fimbul/wotan/src/utils.js:133:25)
    at getFileNamesFromConfigSpecs (/home/travis/build/ioBroker/testing/node_modules/typescript/lib/typescript.js:25928:40)
    at matchFileNames (/home/travis/build/ioBroker/testing/node_modules/typescript/lib/typescript.js:25848:16)
    at getFileNames (/home/travis/build/ioBroker/testing/node_modules/typescript/lib/typescript.js:25430:26)
    at parseJsonConfigFileContentWorker (/home/travis/build/ioBroker/testing/node_modules/typescript/lib/typescript.js:25363:18)
    at Object.parseJsonSourceFileConfigFileContent (/home/travis/build/ioBroker/testing/node_modules/typescript/lib/typescript.js:25323:16)
    at Runner.createPrograms (/home/travis/build/ioBroker/testing/node_modules/@fimbul/wotan/src/runner.js:223:34)
```
Full log: https://travis-ci.org/ioBroker/testing/jobs/526275029

**Comment by ajafff** [comment]
@AlCalzone can you provide the versions of Wotan and TypeScript used.
This should be fixed by https://github.com/fimbullinter/wotan/commit/92e1a4fec68be88bf90428dc2a1496b75b1d84d1 which was released with v0.20.0

**Comment by AlCalzone** [comment]
Yes, on the branch in question, wotan was not updated yet. 0.20.0 fixes the issue.
